src/tarkov-data-manager/jobs/update-quests.js

```diff
@@ -932,8 +932,10 @@ module.exports = async (externalLogger = false) => {
             if (changedQuests[questData.id] && changedQuests[questData.id].objectivesAdded) {
                 for (const newObj of changedQuests[questData.id].objectivesAdded) {
                     if (questData.objectives.some(obj => obj.id === newObj.id)) {
+                        console.log(`${questData.name} skipping objective`,newObj)
```

Comment:
Not sure if these are meant to be left in here but I think its fine either way 🤷‍♂️

Reply:
Nope, note needed to keep